src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/TravelDate/TravelDateSheet.tsx

```diff
@@ -61,8 +67,23 @@ export const TravelDateSheet = forwardRef<ScrollView, Props>(
           ref={focusRef}
           centerContent={true}
         >
+          {maximumDate && (
+            <MessageBox
+              type="info"
+              message={t(
+                TravelDateTexts.latestActivationDate.warning(
+                  formatToVerboseFullDate(maximumDate, language),
+                ),
+              )}
+            />
```

Comment:
Note: Now possible to use the `subtle` flag if we don't want such a "screaming" info box.

Reply:
Good idea, seems to be even more suitable 💡 

Also wrapped the MessageBox in a section to have some padding to the pickers.